### PR TITLE
rephrasing for more logical reading flow

### PR DESCRIPTION
**Pull request by schmalliso**

### CODE REVIEW

File: docs/whatisk8s.md [hunk 1]
```diff
@@ -31,13 +31,8 @@ The *Old Way* to deploy applications was to install the applications on a host u
...
-
-3. With a single application per container, managing the containers becomes tantamount to managing deployment of the application.
+Because containers are small and fast, one application can be packed in each container image. This one-to-one application-to-image relationship unlocks the full benefits of containers. With containers, immutable container images can be created at build/release time rather than deployment time, since each application doesn't need to be composed with the rest of the application stack, nor married to the production infrastructure environment. Generating container images at build/release time enables a consistent environment to be carried from development into production.
+Similarly, containers are vastly more transparent than VMs, which facilitates monitoring and management. This is especially true when the containers' process lifecycles are managed by the infrastructure rather than hidden by a process supervisor inside the container. Finally, with a single application per container, managing the containers becomes tantamount to managing deployment of the application.
```
Comment (schmalliso) [inline docs/whatisk8s.md]: It felt weird to me to list stuff in an ordered list, and then to again list that in an unordered list. The two lists in a row read strangely for me.

**Comment by johndmulhausen** [comment]
@schmalliso Hi Allison, can you please sign the CLA as the googlebot COMMANDS you to do? :) Thanks for the PR!

**Comment by schmalliso** [comment]
@johndmulhausen  I have done so! Thanks!

**Comment by johndmulhausen** [comment]
Thanks, @schmalliso ! If you are interested in $25 of Google Cloud Platform credit, please shoot me an email at johndylan@google.com!

**Pull request merged**